### ISSUE TITLE
Not working column | ad.adsrc does not exist

### ISSUE THREAD

**Issue by micmine** [achiku/planter#11]
**exception**
planter postgres://user:123@localhost/adatabase?sslmode=disable
2019/11/30 16:16:48 failed to get columns of chat: failed to load table def: pq: column ad.adsrc does not exist

**environment**
I am running PostgreSQL inside docker from the official docker [image](https://hub.docker.com/_/postgres) version 12.1-alpine
I have discovered it local and tried it inside the container so docker is probably not the issue.

**Database structure**
It also happens in the example from the repo.

**Comment by rnusser** [comment]
It seems that has been fixed by @mkorolyov in his pull request #10 

So to use it, clone the planter repo, change into the planter directory and:

```
git fetch origin pull/10/head:psql_v12
git checkout psql_v12
go build
./planter  postgres://user:123@localhost/adatabase?sslmode=disable
```

It works for me now using PostgreSQL 12.3 running in a docker container.

**Comment by achiku** [comment]
@rnusser Thanks! It's fixed at #10, so I'll close it :)
